feedback/views.py

```diff
@@ -506,10 +513,10 @@ def get_context_data(self, **kwargs):
         return context
 
 
-class ManageFeedbacksListView(ManageCourseMixin, ListView):
+class ManageFeedbacksListView(ManageCourseMixin, PaginatedMixin, ListView):
     model = Feedback
     template_name = "manage/feedback_list.html"
-    paginate_by = 10
+    paginate_by = 50
```

Comment:
If the teacher has selected a page size in the `paginate_by` select menu, then the selected page size is lost when the teacher makes a new query in the filter feedback form. The teacher needs to select the page size again each time he/she changes the filter parameters. That is frustrating for usability. When the filter feedback form is submitted, could the old `paginate_by` GET query parameter be kept in the URL so that the teacher's chosen page size is not repeatedly forgotten?